### DISCUSSION TITLE
How to Get Server Side authentication context in SSO scenario

### DISCUSSION THREAD

**Discussion by VenkateshSrini** [dotnet/aspnetcore#29225] [Q&A]
Hi,
We want to implement Blazor for an enterprise project. I face a problem with authentication and authorization. The Authentication and authorization happens at the reverse proxy level with IBM Site minder and AD group. When it reaches the app what we get is a JWT. Parsing this JWT with enterprise provided middleware converts the JWT in identity claims that has allowed groups and other identity claims. Now my ask is how do I pass down this server user context to the Blazor. I want this to happen the moment the Blazor web assembly starts.

Please note I use the Blazor web assembly project and **not the** server-side Blazor with SignalR

**Comment by emimontesdeoca** [comment]
There's a great [tutorial](https://jasonwatmore.com/post/2020/08/13/blazor-webassembly-jwt-authentication-example-tutorial) written by @cornflourblue on how to implement JWT Authentication in a Blazor WebAssembly project.
